src/plugins/condition/utils/operations.js

```diff
@@ -253,7 +251,7 @@ export const OPERATIONS = [
             const lhsValue = input[0] !== undefined ? input[0].toString() : '';
             if (input[1]) {
                 const values = input[1].split(',');
-                return values.find((value) => lhsValue === _.trim(value.toString()));
+                return values.find((value) => lhsValue === value.toString().trim());
```

Comment:
These are not directly equivalent. `_.trim()` will tolerate `undefined` as an input, the native approach will not. @shefalijoshi @davetsay is there a reason `_.trim` was used here instead of native trim? Can we safely use the native function here?

Reply:
Will await input from @shefali, @davetsay

Reply:
Looked into this and the lodash method wouldn't help if value was undefined because you can't call toString() on undefined. I'll dig a little deeper.